### DISCUSSION TITLE
Resource polling

### DISCUSSION THREAD

**Discussion by incoming-th** [filamentphp/filament#3467] [Ideas]
Hi,

It seems it is not (yet) possible to have the resource polling - widget supports so why not resource.

https://filamentphp.com/docs/2.x/admin/dashboard/stats#live-updating-polling

Thanks

**Comment by danharrin** [comment]
On the list page?

**Reply by incoming-th** [reply]
Yes correct on the list page.

That's useful for example when models have status that is updated from the jobs and showing in the table view.

Or maybe just create a page with a table widget will have the same behavior?

**Reply by danharrin** [reply]
Okay, this has been added to https://github.com/orgs/filamentphp/projects/2/views/1

**Reply by incoming-th** [reply]
Thanks for considering this!

**Reply by danharrin** [reply]
#3600
